test/integration/controller/jobs/job/job_controller_test.go

```diff
@@ -1086,6 +1086,71 @@ var _ = ginkgo.Describe("Job controller interacting with scheduler", ginkgo.Orde
 		util.ExpectReservingActiveWorkloadsMetric(prodClusterQ, 1)
 	})
 
+	ginkgo.It("Modifying QueueingPolicy", func() {
```

Comment:
This test is pretty much the same as the E2E test. Since integration is cheaper, let's just keep the integration test. There isn't that much value added from the E2E test in this case.

Reply:
okok will remove this test